api/test/shell/cli_test.sh

```diff
@@ -93,3 +93,49 @@ if [[ `grep -c "INFO" ./error.log` -eq '0' ]]; then
     echo "failed: failed to write log on right level"
     exit 1
 fi
+
+
+# etcd basic auth
+# add root user
+curl -L http://localhost:2379/v3/auth/user/add \
+  -X POST -d '{"name": "root", "password": "apisix-dashboard"}'
```

Comment:
A little strange to use this password 🤔

Reply:
so, want more complex like **xdRX3UOis2CY5AWI** or more easy like **root**.
this is just a auth verification test, i think it's all ok.

Reply:
more semantic password like root or password.

Reply:
although i disagreen with this, i also choise modifing it, just to pass it. 😄 😄